### DISCUSSION TITLE
undefined method `pagy_bootstrap_combo_nav_js' for #<ActionView::Base:0x0000000000be50>

### DISCUSSION THREAD

**Discussion by noloman** [ddnexus/pagy#543] [Q&A] [answered]
Hi,

First of all, apologies if this isn't supposed to go here.

I've recently switched from TailwindCSS to Bootstrap and I'm trying to integrate the Javascript Combo Nav `pagy_bootstrap_combo_nav_js` with `Pagy` but I'm always getting:
 
```undefined local variable or method `pagy_bootstrap_combo_nav_js' for #<ActionView::Base:0x0000000000e5d8>`.```
Before, when using TailwindCSS, it was working perfectly but as I switched to Bootstrap, it doesn't work anymore.

I have `<%== pagy_bootstrap_combo_nav_js(@pagy) %>` in my `index.html.erb` and in my `ApplicationHelper` I have:

```
module ApplicationHelper
  include Pagy::Frontend
end
```

Please notice that I haven't followed the instructions of the [Javascript setup][1] as I guess this isn't needed in my case, but I have followed the [Javascript Combo Navs instructions][2].

Thanks in advance!

  [1]: https://ddnexus.github.io/pagy/docs/api/javascript/setup/
  [2]: https://ddnexus.github.io/pagy/docs/api/javascript/combo-navs/

**Comment by benkoshy** [comment, marked as answer]
* Have you required the relevant extra in your pagy.rb initialization file?

```rb
# config/intializlizers/pagy.rb (e.g.)

require 'pagy/extras/bootstrap' # <--- make sure this is here
```

* Pls confirm that you've restarted your server?

If you're using any javascript nav you'd have to set it up:  https://ddnexus.github.io/pagy/docs/api/javascript/setup/


That's all I can think of. 

Ordinarily I'd need a minimal example to replicate, debugging with incomplete information is a trying exercise.

**Reply by noloman** [reply]
wow, you really helped me :) on the first line you wrote the path of `pagy.rb` which should be `config/initializers` in plural, but I had it in `config/initializer` in singular. Just changed the file location and it worked, thanks a lot!

**Reply by BinhTurbo** [reply]
thank you, you really helped me ;)))))
